PokemonRampUp/Model/Sprites.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Sprites.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+// Sprites struct represents multiple properties that contain URL's that direct to Pokemon images
+struct Sprites: Decodable {
+    // There are multiple properties to make sure that atleast one of the poperties is vaild(not nil).
+    let frontDefaultUrl: String?
+    let frontShinyUrl: String?
+    let frontDreamWorldUrl: String?
+    
+    enum CodingKeys: String, CodingKey {
```

Comment:
https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase

Reply:
I added it👍

Reply:
I feel this is business logic. You're deciding what value should be frontDefaultUrl.